### ISSUE TITLE
How can we help?

### ISSUE THREAD

**Issue by steffenix** [mstable/mStable-contracts#131]
How can we help?

I suggest adding issues to the repo for devs that could be interested to help on the project and tag the one that are good for beginners with `good first issue`

**Comment by alsco77** [comment]
Thanks for the suggestion @steffenix , I will do so in the next 48h or so.

**Comment by steffenix** [comment]
ping

**Comment by alsco77** [comment]
Hey @steffenix thanks for pinging me on this. I am super busy atm.. the code on `feeder-pools` has just gone out to audit.

I added a number of tasks to the issues, and tagged some with good first issue. If you are not into any of those, I suggest https://github.com/mstable/mStable-contracts/issues/141 is a valuable/fun one to go for.

Note that most of the pieces of work require a bit more of onboarding. The stuff I have listed here is not related to the CORE protocol, but would certainly be helpful if done. There are a number of larger pieces of work coming up in the next few months.. i'll try to break things off where I can.

Feel free to open issues if you feel that for example the documentation or test suite could be improved.
